server/lib/convert-article.js

```diff
@@ -25,31 +22,42 @@ module.exports = exports = ({ source, sourceFormat = 'html', targetFormat = 'doc
 		let cmd = spawn(CONVERT_FORMAT_COMMAND, args);
 
 		let targetBuffer = Buffer.from('', 'utf8');
-		let error = '';
+		let stderr = '';
 
-		cmd.on('error', err => {
-			log.error(`${MODULE_ID} ${CONVERT_FORMAT_COMMAND} ConversionError`, {
-				error: err.stack
+		cmd.on('error', error => {
+			log.error({
+				cmd: CONVERT_FORMAT_COMMAND,
+				event: 'CONVERSION_ERROR',
+				error
 			});
 
-			reject(err);
+			reject(error);
 		});
 
-		cmd.stdout.on('data', chunk =>
-			targetBuffer = Buffer.concat([targetBuffer, chunk]));
+		cmd.stdout.on('data', chunk => {
+			targetBuffer = Buffer.concat([targetBuffer, chunk])
```

Comment:
this does change functionality, not sure if the function needs to return something?